### PR TITLE
add if outgoing payment request, remove header title with balance

### LINKED ISSUE #21: Remove balance display from Payment Request screen
### Expected Behavior
Balance is not visible on the Payment Request Enter Amount screen.

### Current Behavior
Balance is visible, which leads users to believe it's relevant to requesting payments.

https://files.slack.com/files-pri/T9SEZ6M9B-F01N3N1BS3G/image_from_ios.png

### PR DESCRIPTION
**Pull request by michellewong793**
### Description

Changes the SendScreen component: 
Header is without the balance on the Request Payments Screen. 

image

### Other changes

none 

### Tested

tested on android 

### Related issues

- Fixes #21
- https://app.zenhub.com/workspaces/wallet-backlog-board-60092d006a84e3001d0d5936/issues/celo-org/wallet/21

### Backwards compatibility

compatible

### CODE REVIEW

**Review by etuleu — APPROVED** [review]
Awesome!

File: packages/mobile/src/send/SendAmount.tsx [hunk 1]
```diff
@@ -87,7 +87,12 @@ export const sendAmountScreenNavOptions = ({
   return {
     ...emptyHeader,
     headerLeft: () => <BackButton eventName={eventName} />,
-    headerTitle: () => <HeaderTitleWithBalance title={title} token={CURRENCY_ENUM.DOLLAR} />,
+    headerTitle: () =>
+      route.params?.isOutgoingPaymentRequest ? (
+        <HeaderTitleWithSubtitle title={title} subTitle={' '} />
```
Comment (gnardini) [inline packages/mobile/src/send/SendAmount.tsx:92]: In this case we're using a `HeaderTitleWithSubtitle` with an empty subtitle. I think it makes more sense to just pass the title directly if it's a payment request. It doesn't even need to be a function, you can do something like:
```
headerTitle: isOutgoingPaymentRequest ? title : () => (<HeaderTitleWithBalance ...>)
```

Comment (jeanregisser) [reply packages/mobile/src/send/SendAmount.tsx:92]: +1, with the empty subtitle, the vertical alignment of the title is not right.

Comment (michellewong793) [reply packages/mobile/src/send/SendAmount.tsx:92]: thanks! changed

**Review by gnardini — APPROVED** [review]
Awesome! 🙌

**Comment by Celoqa** [comment]
Hi @etuleu  I have verified this issue using latest released Android internal build V1.14.0 (1004294350) & Test Flight build V1.14.0(58) & observed following:
**Observations:**

- [X] It is observed that, user is able to see Header without the balance on the "Request Payments Screen" using phone number & QR code of User B when user has some balance in account or $0.00 account balance. [Attachment](https://user-images.githubusercontent.com/79188345/117613634-d149b480-b184-11eb-9016-7efcb765dbb0.png)

**For Verified Phone Number:**

- [ ] It is observed that, user able to see "Please confirm your number before sending cUSD to a contact" red validation message displayed when user requesting money with verified phone number using Request Contact Screen. [Attachment](https://drive.google.com/file/d/1LKHXqFPHFNHal6xJbESApw2P7ZWdk79t/view?usp=sharing)

- [X] Also it is observed that, user able to request completely even user has $0.00 balance in verified user's acount without any error message through scanning QR code of user B.

**For Non-verified Number:**

- [ ] It is observed that while requesting through non verified number  using Request Contact screen "Please confirm your number before sending cUSD to a contact" red validation message displayed & loading indicator displayed for longer time even user proceeds by tapping on "Review" button. [Attachment](https://drive.google.com/file/d/1RaGR_dbG0jK4_VN8pJmFuP_RsEnQ0DmM/view?usp=sharing)
Let me know is this expected or we need to raise a ticket for same ?

- [X] Also it is observed that, user able to request completely even user has $0.00 balance for non-verified user without any error message through scanning QR code of user B. [Attachment](https://drive.google.com/file/d/1tyuYm0NyViOqBs38E04ZxSfwuFXq3g8x/view?usp=sharing)


**Verified On Devices:** iPhone SE (14.2), Redmi Note 8 (9.0), iPhone 6+(12.4.5)
Thanks..!!

**Comment by Celoqa** [comment]
Hey @etuleu Is there any update on above unchecked points ?

**Pull request merged**